src/light_client.rs

```diff
@@ -317,27 +317,26 @@ pub async fn process_block(
 
 		info!(
 			block_number,
-			"partition_dht_rows_insert_time_elapsed" = time_elapsed,
+			"partition_dht_rows_insert_time_elapsed" = ?time_elapsed,
```

Comment:
Does prints milliseconds instead of seconds?

Reply:
It prints proper time like `1s` or `20ns` or combined